base/BUILD.gn

```diff
@@ -3132,7 +3132,8 @@ test("base_unittests") {
     "big_endian_unittest.cc",
     "bit_cast_unittest.cc",
     "bits_unittest.cc",
-    "build_time_unittest.cc",
+    # Cobalt builds don't include build timestamps.
+    # "build_time_unittest.cc",
```

Comment:
We should either have a conditional checking use_cobalt_customizations that removes the entire source file if we don't want any of it or we should add the DISABLED_ gtest macro if we're disabling just certain tests in the file, that way we can have easier ways of tracking disabled tests